registry/mainnet/interchain/squid.tokenlist.json

```diff
@@ -63,6 +63,46 @@
           "name": "RMRK"
         }
       ]
+    },
+    "0x6dac9057eb38cc61358b1b25453e3eb68a0dce959d8549dd4b8e58f357be0f66": {
+      "tokenId": "0x6dac9057eb38cc61358b1b25453e3eb68a0dce959d8549dd4b8e58f357be0f66",
+      "deployer": "0x6E2A0A942Eaa01B7EB2639B8B65D1D2768a87BAc",
+      "originalMinter": null,
+      "prettySymbol": "FS",
+      "decimals": 18,
+      "originAxelarChainId": "fantom",
+      "tokenType": "canonical",
+      "deploySalt": "0x",
+      "iconUrls": {
+        "svg": "https://raw.githubusercontent.com/axelarnetwork/axelar-configs/images/tokens/fs.svg"
```

Comment:
minor nit, but can you change this to `https://raw.githubusercontent.com/axelarnetwork/axelar-configs/main/images/tokens/fs.svg`

we're about to make a change to the scripts so that future projects autogenerate with that main prefix.

also, can you actually upload an `fs.svg` if you plan for us to host one there?

Reply:
![fs](https://github.com/axelarnetwork/axelar-configs/assets/87469644/b555446a-f80b-45fb-8a48-7f398ecded0f)